lib/diagnostic.rb

```diff
@@ -108,10 +108,12 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'person = {:favorite_number => 42, :first_name => "joe"}
```

Comment:
Remember that we can shorten these even more:

`'person = {favorite_number: 42, first_name: "joe"}`